SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduledJobPoller.java

```diff
@@ -112,10 +112,20 @@ public void runActionOnPoll() {
 
       final CronExpression cronExpression;
 
+      Optional<String> maybeSchedule = request.getRequest().getQuartzScheduleOrSchedule();
+      if (!maybeSchedule.isPresent()) {
```

Comment:
Hmm, the entire point of the safe() method is to return a known non-null value. The fact that it's possible to return a null value from there indicates a broken assumption elsewhere that I think we should fix. Otherwise, we can't have nice things anywhere and have to constantly question all non-optional returns.